cfgov/cfgov/settings/base.py

```diff
@@ -0,0 +1,89 @@
+import os
+from unipath import Path
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+CFGOV_REFRESH = Path(__file__).ancestor(5).child('cfgov-refresh')
```

Comment:
Yeah I realized the parent() thing during lunch /facepalm :worried: 

Should we create a PROJECT_ROOT setting instead and then get the parent from that? Seems like we'd find putting the settings file where we want useful?